prebid-mobile/pbm-api/iOS/pbm-api-iOS.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page_v2
+title: Prebid Mobile API - iOS
+description: Prebid Mobile API - iOS
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile
+sidebarType: 2
+---
+
+# Prebid Mobile API - iOS 
+
+Use the Prebid Mobile API 1.0 for iOS to implement header bidding in your mobile apps. The elements in the API will allow you to participate in a header bidding auction and communicate with your ad server to display a creative. The API supports banner and interstitial creatives. 
+
```

Comment:
Key features for the new API's would be
1. Publisher knows if the keywords are attached to the adUnit or not
2. PBM now implements and supports its own auto refresh and doesn't support adServers refresh anymore.(we had done this previously)
3. Clear result codes that details the result of the prebid demand fetch request